SampleTest/Package.appxmanifest

```diff
@@ -7,15 +7,15 @@
   IgnorableNamespaces="uap mp">
 
   <Identity
-    Name="54b2f5a5-a4ed-4158-b816-d7a6a07d85e5"
-    Publisher="CN=mhawker"
+    Name="1206Weaver.GraphControlsTestApp"
```

Comment:
Assuming this was from the store association step? Just wanted to check as we'll need to document that somewhere, eh?

Reply:
Hehe, I should have left this out. Ya, I did an association to get the auth to work. We will need to document this, because it fails without it.